evap/student/templates/student_vote.html

```diff
@@ -132,6 +132,30 @@ <h3 class="mb-3">{{ course.name }} ({{ course.semester.name }})</h3>
             // (the data gets deleted every time the form is submitted, i.e. also when the form had errors and is displayed again)
             sisyphus.saveAllData();
 
+            // Show a warning on textareas if they only contain obsolete text (punctuation, k.A., n/a, ...)
+            function isTextareaInvalid(textarea) {
+                let val = textarea.val();
+                return /^\W+$/g.test(val) || ["ka", "na"].indexOf(val.toLowerCase().replace(/\W/g,'')) != -1;
```

Comment:
instead of the first part of the or you can add an empty string to the array, this way there's not a second regex to parse for the reader. then `textarea.val()` can be inlined. also there's a nicer [includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes) method, and you could put the `toLowerCase` behind the `replace` because there's no need to lowercase punctuation, can you try that?

Reply:
what do you mean by empty val? an empty string? that should just work...

Reply:
ah, *exclude*, now i understand.